fix grep string for check 1.4.11 and 1.4.12

hutr opened the pull request:
check 1.4.11 and 1.4.22 shows FAIL even when permissions is correct.

lizrice commented on cfg/1.8/master.yaml:
```diff
@@ -942,9 +942,8 @@ groups:
-    tests:
-      test_items:
+    audit: ps -ef | grep $etcdbin | grep -- --data-dir | grep -v grep | sed 's%.*data-dir[= ]\([^ ]*\).*%\1%' | xargs stat -c %a
+    test_items:
```
Looks like this is missing `tests:`?

hutr replied:
Added tests:

lizrice commented on cfg/1.8/master.yaml:
```diff
@@ -942,9 +942,8 @@ groups:
-    tests:
-      test_items:
+    audit: ps -ef | grep $etcdbin | grep -- --data-dir | grep -v grep | sed 's%.*data-dir[= ]\([^ ]*\).*%\1%' | xargs stat -c %a
+    test_items:
```
I think this is missing `tests:`

lizrice commented on cfg/1.8/master.yaml:
```diff
@@ -942,9 +942,8 @@ groups:
-    audit: ps -ef | grep $etcdbin | grep -v grep | sed 's%.*data-dir[= ]\([^ ]*\).*%\1%' | xargs stat -c %a
-    tests:
-      test_items:
+    audit: ps -ef | grep $etcdbin | grep -- --data-dir | grep -v grep | sed 's%.*data-dir[= ]\([^ ]*\).*%\1%' | xargs stat -c %a
```
I don't think there is any need for the grep -v grep step now that we are checking for --data-dir

hutr replied:
You are correct about this. Removed grep -v grep

lizrice commented on cfg/1.8/master.yaml:
```diff
@@ -960,7 +959,7 @@ groups:
   - id: 1.4.12
     text: "Ensure that the etcd data directory ownership is set to etcd:etcd (Scored)"
-    audit: ps -ef | grep $etcdbin | grep -v grep | sed 's%.*data-dir[= ]\(\S*\)%\1%' | xargs stat -c %U:%G
+    audit: ps -ef | grep $etcdbin | grep -- --data-dir | grep -v grep | sed 's%.*data-dir[= ]\(\S*\)%\1%' | xargs stat -c %U:%G
```
I think grep -v grep  is now unnecessary here too

hutr replied:
Removed grep -v grep

Thanks for correction!

hutr commented:
Thanks for the correction.

Added tests: for 1.4.11
Removed grep -v grep for 1.4.11 and 1.4.12
Also fixed sed string for 1.4.12 to fetch right --data-dir folder
